docs/snapshot-appr-registry.md

```diff
@@ -0,0 +1,152 @@
+
+- a point-in-time export of an Appregistry (appr) type catalog's content
+- the result of converting an appr catalog to a container image type catalog
+- an immutable artifact
+
+## When/Why would I need a snapshot?
+
+OLM always installs from the latest version of an appr catalog. As appr catalogs are updated, the latest versions of operators change, and older versions may be removed or altered. This behavior can cause problems maintaining reproducible installs over time.
+
+As of OCP 4.3, Red Hat provided operators are distributed via appr catalogs. Creating a snapshot provides a simple way to use this content without incurring the aforementioned issues. 
+
+## Prerequisites
+
+- Linux
+- A version `oc` that has the `oc catalog build`, such as [this one for OCP `4.3`](https://openshift-release-artifacts.svc.ci.openshift.org/4.3.0-0.nightly-2019-11-13-233341/openshift-client-linux-4.3.0-0.nightly-2019-11-13-233341.tar.gz)
+- A container image registry that supports [Docker v2-2](https://docs.docker.com/registry/spec/manifest-v2-2/)
+- [grpcurl](https://github.com/fullstorydev/grpcurl) (optional, for testing)
+
+## Setup
+
+Users should authenticate with the target image registry. By default, `oc adm catalog build` uses `~/.docker/config.json` to determine credentials. 
+
+## Taking a snapshot
+
+Snapshot `redhat-operators` as `quay.io/my/redhat-operators:snapshot-0`.
+
+```sh
+$ oc adm catalog build --appregistry-endpoint https://quay.io/cnr --appregistry-org redhat-operators --to=quay.io/my/redhat-operators:snapshot-0
+INFO[0013] loading Bundles                               dir=/var/folders/st/9cskxqs53ll3wdn434vw4cd80000gn/T/300666084/manifests-829192605
+INFO[0013] directory                                     dir=/var/folders/st/9cskxqs53ll3wdn434vw4cd80000gn/T/300666084/manifests-829192605 file=manifests-829192605 load=bundles
+INFO[0013] directory                                     dir=/var/folders/st/9cskxqs53ll3wdn434vw4cd80000gn/T/300666084/manifests-829192605 file=3scale-operator load=bundles
+INFO[0013] found csv, loading bundle                     dir=/var/folders/st/9cskxqs53ll3wdn434vw4cd80000gn/T/300666084/manifests-829192605 file=3scale-operator.v0.3.0.clusterserviceversion.yaml load=bundles
+INFO[0013] loading bundle file                           dir=/var/folders/st/9cskxqs53ll3wdn434vw4cd80000gn/T/300666084/manifests-829192605/3scale-operator file=3scale-operator.package.yaml load=bundle
+INFO[0013] loading bundle file                           dir=/var/folders/st/9cskxqs53ll3wdn434vw4cd80000gn/T/300666084/manifests-829192605/3scale-operator file=3scale-operator.v0.3.0.clusterserviceversion.yaml load=bundle
+...
+Uploading ... 244.9kB/s
+Pushed sha256:f73d42950021f9240389f99ddc5b0c7f1b533c054ba344654ff1edaf6bf827e3 to quay.io/my/redhat-operators:snapshot-0
+```
+
+Sometimes bad manifests are accidentally introduced into Red Hat's catalogs, when this happens you may see some errors.
```

Comment:
Is this sentence necessary? If so, maybe `Sometimes invalid manifests` ?

Reply:
I think it gives context to why I'm showing an error on the next line. This also happens a lot, for this reason, so I thought this would save us from answering a support question in the future.

> If so, maybe Sometimes invalid manifests ?

Will update it soon!